### ISSUE TITLE
[Bug] rotation_matrix_to_angle_axis produces wrong result

### ISSUE THREAD

**Issue by d-walsh** [kornia/kornia#638]
## To Reproduce

If I run the code:
**rotation_matrix** is different to **new_rotation_matrix**

```
rotation_matrix = # valid rotation matricies
angle_axis = rotation_matrix_to_angle_axis(rotation_matrix)
new_rotation_matrix = angle_axis_to_rotation_matrix(angle_axis)
```

## Expected behavior

I expect `rotation_matrix_to_angle_axis` to return a correct result.

## Environment

I am using Korina **0.4.0** installed via pip

 - PyTorch Version (e.g., 1.0): 1.6.0
 - OS (e.g., Linux): Ubnutu
 - How you installed PyTorch (`conda`, `pip`, source): pip
 - Build command you used (if compiling from source):
 - Python version: 3.7
 - CUDA/cuDNN version: 11
 - GPU models and configuration:
 - Any other relevant information:

**Comment by takuya-ikeda-tri** [comment]
Hi @edgarriba, @krrish94, thank you for the awesome library.
I got the same problem. What is the current status?

**Comment by edgarriba** [comment]
@takuya-ikeda-tri no one working on this issue at this moment. Contributions are very welcomed

**Comment by d-walsh** [comment]
I used the implementation from https://github.com/utiasSTARS/liegroups and it works well.

**Comment by edgarriba** [comment]
there's a pending integration of liegroups from @krrish94 in staled sfm branch
https://github.com/kornia/kornia/commit/13614e81fe84d61b09e0c9f42b205247760c6fa0

**Comment by edgarriba** [comment]
This will be fixed after https://github.com/kornia/kornia/issues/483

**Comment by edgarriba** [comment]
fixed in #926
